tests/integration_test_net_test.go

```diff
@@ -198,10 +198,10 @@ func TestIntegrationTestNet_CanStartWithCustomConfig(t *testing.T) {
 	// validation behaves as expected.
 	net := StartIntegrationTestNet(t, IntegrationTestNetOptions{
 		ModifyConfig: func(config *config.Config) {
-			// enable minimum tip check for local tx submission
+			// disable minimum tip check for non-local tx submission
 			config.TxPool.NoLocals = true
```

Comment:
I think the original comment was correct.
Settings `NoLocals` to true tells the pool to handle local transactions the same way it handles remotes - which should enables checks for local txs, as the original comment claim.